src/TextInput.tsx

```diff
@@ -41,18 +50,22 @@ const TextInput = React.forwardRef<HTMLInputElement, TextInputInternalProps>(
       <TextInputWrapper
         block={block}
         className={wrapperClasses}
+        status={status}
```

Comment:
Should this be called `validationStatus` to be consistent with @rezrah's new checkbox and radio components?

Reply:
Got it!